pipeline_steps/common.groovy

```diff
@@ -22,10 +22,11 @@ def install_ansible(){
     source .venv/bin/activate
 
     # These pip commands cannot be combined into one.
-    pip install -U six packaging appdirs
-    pip install -U setuptools
     pip install 'pip==9.0.1'
+    pip --isolated install -U six packaging appdirs
```

Comment:
this install should use constraints for pinning - not sure why they're not in requirements/constraints?